neurobooth_os/tasks/task.py

```diff
@@ -88,7 +88,7 @@ def __init__(
 
         if self.path_instruction_video is not None:
             self.path_instruction_video = op.join(
-                cfg.neurobooth_config["video_tasks"], self.path_instruction_video
+                cfg.neurobooth_config.video_task_dir, self.path_instruction_video
```

Comment:
I ended up adding this change in my PR as well, will resolve conflict when it arises :)